scripts/cli/tests/4_permission_management.js

```diff
@@ -42,13 +72,136 @@ async function main() {
   process.exit();
 }
 
-async function setPermissionToSigner(api, accounts, secondary_accounts) {
+function setDoc(docArray, uri, contentHash, name, docType, filingDate) {
+  docArray.push({
+    "uri": uri,
+    "content_hash": contentHash,
+    "name": name,
+    "doc_type": docType,
+    "filing_date": filingDate
+  });
+}
+
+async function addDocuments(api, ticker, docs, signer) {
+  try {
+    const transaction = api.tx.asset.addDocuments(docs, ticker);
+    await reqImports.sendTx(signer, transaction);
+    return true;
+  } catch(err) {
+    return false;
+  }
+}
+
+async function movePortfolioFunds(api, primary_key, secondary_key, ticker, amount) {
+  try {   
+    const primaryKeyDid = await reqImports.getDid(api, primary_key);
+    const secondaryKeyDid = await reqImports.getDid(api, secondary_key);
+    const portfolioNum = (await nextPortfolioNumber(api, secondaryKeyDid)) - 1;
+
+    const from = {
+      did: primaryKeyDid,
+      kind: 'Default'
+    }
+    
+    const to = {
+      did: secondaryKeyDid,
+      kind: {User: portfolioNum}
+    }
+    
+    const items = [
+      {
+        ticker,
+        amount
+      }
+    ]
+    
+    const transaction = api.tx.portfolio.movePortfolioFunds(from, to, items);
+    await reqImports.sendTx(secondary_key, transaction);
+    return true;
```

Comment:
Instead of just sending the tx, we should also make sure that the action happened (funds moved in this case).

Even if permissions were missing, the tx will be successful but the funds will not be moved.

Reply:
Alright made the change.